grails-app/views/header/_mainPortalHeader.gsp

```diff
@@ -1,13 +1,17 @@
 <div id="header">
 
     <div id="logoContainer">
-        <a href="landing"><img src="${portalBranding.logoImage}" alt="logo" id="logo" />
+        <a href="landing"><img src="${portalBranding.logoImage}" alt="main logo" id="logo" height="80" />
         </a>
     </div>
     <div id="landingHeaderContainer" class="headerHeightOverlord">
         <h1 id="headerTitle">${portalBranding.siteHeader}</h1>
     </div>
-
+    <g:if test="${portalBranding.secondaryLogoImage}">
```

Comment:
I'd recommend to not having such logic in a gsp, but rather have `portalBranding` return the default value, which can be just a blank image, or something similar. The logic already exists there and it's one less control path to test.